webpack.config.prod.js

```diff
@@ -29,7 +32,7 @@ module.exports = {
         loader: 'babel-loader',
       },
       {
-        test: /\.scss$/,
+        test: /\.(scss|css)$/,
```

Comment:
I recommend adding a different loader just for CSS, that won't include `sass-loader` in the `use` array below.